codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/RustWriter.kt

```diff
@@ -328,7 +328,7 @@ class RustWriter private constructor(
     private val commentCharacter: String = "//",
     private val printWarning: Boolean = true,
     /** Insert comments indicating where code was generated */
-    private val debugMode: Boolean = false,
+    private val debugMode: Boolean = true,
```

Comment:
This should be reverted, it's causing the unit tests to fail.

Reply:
Yep, pushed to get a link to CI in the case where it's `true`. Reverting it now.